execution-plan-traits/src/lib.rs

```diff
@@ -51,16 +51,28 @@ impl From<Address> for InMemory {
     }
 }
 
+/// Select a memory area.
+/// Intended to use for storing return values.
+#[derive(Clone, Copy, Serialize, Deserialize, PartialEq, Eq, Hash, Debug)]
+pub enum MemoryArea {
```

Comment:
Already have an `enum Destination` for this: https://github.com/KittyCAD/modeling-api/blob/f38096b3893fd36c9792216f3b87ced2254d2d04/execution-plan/src/lib.rs#L34-L41